src/API/Helpers/State/DummyStateHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Auth0\SDK\API\Helpers\State;
+
+/*
+ * This file is part of Auth0-PHP package.
+ *
+ * (c) Auth0
+ *
+ * For the full copyright and license information, please view the LICENSE file
+ * that was distributed with this source code.
+ */
+
+/**
+ * This interface must be implemented by state handlers.
+ *
+ * @author Auth0
+ */
+class DummyStateHandler implements StateHandler 
+{
+    /**
+     * Generate state value to be used for the state param value during authorization.
+     * 
+     * @return string
+     */
+    public function issue() {
+        return null;
+    }
+
+    /**
+     * Store state value to be used for the state param value during authorization.
+     * 
+     */
+    public function state($state) {
```

Comment:
Typo on the method: Should be `store` (based on the interface)

Reply:
thx, this is why we lots of testing in 6.x 😄